WcaOnRails/spec/models/user_spec.rb

```diff
@@ -345,7 +345,8 @@
       other_person = FactoryGirl.create :person, year: 1980, month: 2, day: 1
       user.unconfirmed_wca_id = other_person.wca_id
       expect(user).to be_invalid
-      expect(user.errors.messages[:dob_verification]).to eq ['incorrect']
+      expect(user.errors.messages[:dob_verification].length).to eq 1
+      expect(user.errors.messages[:dob_verification][0]).to start_with "The provided date of birth doesn't match our database."
```

Comment:
Hmm... one easy fix in this case is to expect `I18n.t('users.errors.dob_incorrect_html')`. I think that's better than what we have now. Let me know what you think!